src/Minicomponents/InputField.jsx

```diff
@@ -0,0 +1,29 @@
+export default function Input({
+  label,
+  type,
+  value,
+  onChange,
+  placeholder,
+  name,
+  id,
+  required = false,
+  className = "",
+  ...props
+}) {
+  return (
+    <div className={className}>
+      {label && <label>{label}</label>}
+      <input
+        id={id || name}
+        type={type}
+        value={value}
+        onChange={onChange}
+        placeholder={placeholder}
+        name={name}
+        required={required}
+        className={className}
+        {...props}
```

Comment:
height and width can also be changed , so we can add that too

Reply:
Sure! I'll update the Input component to include customizable height and width as props.

Reply:
I've updated the Input component to include customizable height and width as props. Let me know if there's anything else you'd like to add or modify!

Reply:
wait